### PR TITLE
SCAP Tailoring File entity

### PR DESCRIPTION
**Pull request by tstrych**
Adding SCAP Tailoring File entity with view. Please review @abalakh or @oshtaier .

I didn't push test yet, because some of them are connected with oscap policy entity.
QUESTION: what about stubbed tier2,tier3,tier4 test ? Do I port them ? Do I delete them from ui?

### CODE REVIEW

File: airgun/entities/oscaptailoringfile.py [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+    SCAPTailoringFileCreateView,
+    SCAPTailoringFileEditView,
+    SCAPTailoringFilesView,
+)
+
+
+class OSCAPTailoringFileEntity(BaseEntity):
+
+    def create(self, values):
+        """Creates new SCAP
```
Comment (oshtaier) [inline airgun/entities/oscaptailoringfile.py]: `Creates new SCAP`?

File: airgun/views/oscaptailoringfile.py [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+    @View.nested
+    class organizations(SatTab):
+        resources = MultiSelect(id='ms-tailoring_file_organization_ids')
+
+
+class SCAPTailoringFileEditView(SCAPTailoringFileCreateView):
+
+    @property
+    def is_displayed(self):
```
Comment (oshtaier) [inline airgun/views/oscaptailoringfile.py]: maybe we need to override file upload tab to also be able to read already uploaded file value:
image

File: airgun/views/oscaptailoringfile.py [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+        breadcrumb_loaded = self.browser.wait_for_element(
+            self.breadcrumb, exception=False)
+        return (
+                breadcrumb_loaded
+                and self.breadcrumb.locations[0] == 'Tailoring files'
+                and not self.breadcrumb.read().starts_with('Edit ')
```
Comment (abalakh) [inline airgun/views/oscaptailoringfile.py]: why `and not ...starts_with('Edit ')` for Edit view? Maybe `and ... starts_with()` instead? Or, as a last resort option, `and self.breadcrumb.read() != 'Upload new Tailoring File'`

**Comment by tstrych** [comment]
@oshtaier  or @abalakh  please review. 
Also please take a look at [test](https://github.com/SatelliteQE/robottelo/pull/6239)

**Comment by oshtaier** [comment]
`and self.breadcrumb.read().starts_with('Edit ')`

File: airgun/views/oscaptailoringfile.py [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+    class locations(SatTab):
+        resources = MultiSelect(id='ms-tailoring_file_location_ids')
+
+    @View.nested
+    class organizations(SatTab):
+        resources = MultiSelect(id='ms-tailoring_file_organization_ids')
+
+
+class SCAPTailoringFileEditView(SCAPTailoringFileCreateView):
+    scap_file_name = Text("//b")
```
Comment (oshtaier) [inline airgun/views/oscaptailoringfile.py]: is it unique element? Like there is no other elements in DOM with `//b` tag?

Comment (tstrych) [reply airgun/views/oscaptailoringfile.py]: `and self.breadcrumb.read().starts_with('Edit ')`
But it does't start with `Edit`

**Comment by tstrych** [comment]
@oshtaier or @abalakh look at this [post](https://github.com/SatelliteQE/airgun/pull/166#issuecomment-417086822)

**Comment by oshtaier** [comment]
rebase required

**Comment by tstrych** [comment]
@oshtaier  rebased, and your xpath used.

**Pull request merged**